src/endpoint/endpoint.js

```diff
@@ -231,12 +231,14 @@ async function main(options = {}) {
         if (internal_rpc_client && config.NAMESPACE_MONITOR_ENABLED) {
             endpoint_stats_collector.instance().set_rpc_client(internal_rpc_client);
 
-            // Register a bg monitor on the endpoint
-            background_scheduler.register_bg_worker(new NamespaceMonitor({
-                name: 'namespace_fs_monitor',
-                client: internal_rpc_client,
-                should_monitor: nsr => Boolean(nsr.nsfs_config),
-            }));
+            setTimeout(() => {
```

Comment:
why not add a retry in the namespace monitor if the error was ENOENT and start time of the endpoint is lower than 60 seconds? also why 60 seconds?

Reply:
Sounds like more code with equivalent solution.

60 seconds to allow pod to stabilize (get to 'ready' or be deleted).

Reply:
the reason is understood, but how do you know it takes a minute? is it always a minute? should it be configurable?

Reply:
AFAIK, there's no way for node to know status of the pod.
I assume we don't want to make such a dependency.

60 seconds is more than enough on my minikube and is low enough not to bother other deployments in other envs, but I can make it an env variable.